src/Terms.ts

```diff
@@ -27,6 +27,10 @@ export class TermsNotSignedError extends Error {}
  * require agreement from the user before the user can use that service.
  */
 export class Service {
+    public serviceType: string;
+    public baseUrl: string;
+    public accessToken: string;
```

Comment:
these could be inlined into c'tor as

`constructor(public serviceType: string, public baseUrl: string, public accessToken: string) {}`